Makefile

```diff
@@ -435,17 +439,38 @@ codecoverage: ## Generate Code Coverage report for Apollo tests
                 "./scripts/run-codecoverage.sh"
 	@echo "Completed make codecoverage"
 
-BUILD_IMAGE_MODE=ALL
+# base docker image with toolchain only
+GIT_BRANCH=$(shell git name-rev --name-only HEAD | sed "s/~.*//")
```

Comment:
Makefile now depends on git being available in the host, which is not a reasonable assumption for people who want just to test the (opensource) repository.
Need to add default empty values.

Reply:
which values?

Reply:
fixed